runtime/src/accounts_index_storage.rs

```diff
@@ -167,11 +167,11 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> AccountsIndexStorage<
             .collect::<Vec<_>>();
 
         Self {
-            _bg_threads: BgThreads::new(&storage, &in_mem, threads, true, exit),
+            _bg_threads: BgThreads::new(&storage, &in_mem, threads, true, &exit),
```

Comment:
Calling this out - I left this one (the parameter to `BgThreads::new()`) as a reference, since we end up cloning for each worker thread within that function.

Supposing we changed to pass `Arc` instead of `&Arc`, the simple approach of cloning in the loop would result in one extra clone. Ie, we'd clone once to pass into the function and once for each thread. Could technically get around this, but opted to leave this one as is.